app/Importers/WindDataImporter.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Importers;
+
+use GuzzleHttp\Exception\GuzzleException;
+use Symfony\Component\DomCrawler\Crawler;
+
+class WindDataImporter extends DataImporter
```

Comment:
It would be good to write tests for this importer.

Reply:
How this test should look like, and what it should test?

Reply:
My suggestion is to leave this test for later - after yesterday's conversation, I agreed that I would test each provider

Reply:
As you wish, but create draft PR with at least one importer tested, and ping me or someone else for a review.
Suggested way of creating new features is to write tests for them in the same PR.